File: .net & mono/Weed3Demo/demo_plus/storeProcedure/user_update.cs
using Noear.Weed;
using System;
using System.Data;

namespace Weed3Demo.demo_ex {
    public class user_update : DbStoredProcedure {
        public user_update() : base(DbConfig.test) {
            call("user_update");
            set("_userID",() => userID);
            set("_city", () => city);
            set("_vipTime", () => vipTime);
        }

        public long userID;
        public string city;
        public DateTime vipTime;
    }
}

File: java/weed3-test/src/main/java/weed3demo/demo_plus/storeProcedure/user_update.java
package weed3demo.demo_plus.storeProcedure;

import org.noear.weed.DbQueryProcedure;
import org.noear.weed.DbStoredProcedure;
import weed3demo.config.DbConfig;

import java.util.Date;

/**
 * Created by noear on 2017/7/22.
 */
public class user_update extends DbStoredProcedure {
    public user_update() {
        super(DbConfig.test);

        call("user_update");
        set("_userID", () -> userID);
        set("_city", () -> city);
        set("_vipTime", () -> vipTime);
    }

    public long userID;
    public String city;
    public Date vipTime;
}
